x/payment/types/builder.go

```diff
@@ -67,12 +67,17 @@ func (k *KeyringSigner) NewTxBuilder() sdkclient.TxBuilder {
 // BuildSignedTx creates and signs a sdk.Tx that contains the provided message. The interal
 // account number must be set by calling k.QueryAccountNumber or by manually setting it via
 // k.SetAccountNumber for the built transactions to be valid.
-func (k *KeyringSigner) BuildSignedTx(builder sdkclient.TxBuilder, msg sdktypes.Msg) (authsigning.Tx, error) {
+func (k *KeyringSigner) BuildSignedTx(msg sdktypes.Msg, options ...TxBuilderOption) (authsigning.Tx, error) {
```

Comment:
this slight refactor isn't really inside the scope of this PR, I decided to include it anyway, as it is very minor.

Reply:
can we pull this out or can we add this into the pr description. In the future if we need to reference this PR for this code it could get confusing.

Reply:
pulled this out, and will add it in a different PR